modules/codemod/lib/v6/spec/deprecateHeader.spec.ts

```diff
@@ -8,7 +8,99 @@ describe('Canvas Kit Deprecate Header Codemod', () => {
   context('when transforming header imports', () => {
     it('should ignore non-canvas-kit imports', () => {
       const input = `import Header, {GlobalHeader} from "@workday/some-other-library";`;
-      const expected = `import Header, {GlobalHeader} from "@workday/some-other-library";`;
+      const expected = `${input}`;
+
+      expectTransform(input, expected);
+    });
+
+    it('should ignore non-canvas-kit header identifiers', () => {
```

Comment:
Please review the new test cases to see if I covered anything we can think of. `Header` is the most common component name that conflicts with non-Canvas Kit components. We could have more general spec files to test the utilities, but I hope this is sufficient for now.